latex/Research_Strategy.tex

```diff
@@ -167,17 +169,20 @@
+Governance transitioned to an unfunded academic collaboration during SAMPL3 in 2012, who ran subsequent challenges as SAMPL4 (2014) and SAMPL5 (2016).
+The PI of this proposal (Mobley) played a key organizational role in SAMPL3--SAMPL5.
+SAMPL is modeled as a \emph{challenge} rather than a \emph{competition}, with a key goal being to maximize what is learned rather than to declare winners and losers, with the idea that learning collectively from failures and successes will provide the greatest long-term rewards in terms of progress in the field.
 SAMPL has always involved a component focused on calculation of relatively straightforward physical properties such as hydration free energies, but also introduced host-guest systems as model binding systems for SAMPL3-5, supplementing protein-ligand challenges (on trypsin and HIV integrase) which appeared in SAMPL3-4. 
 SAMPL has already been a tremendous resource for the community, resulting in roughly 100 publications (some are coming out as of this writing) which are typically cited 5-50 times or more each [refs].
+{\color{red}[JDC: We need to rework this paragraph to convey the main idea of \emph{why} this specific kind of competition is able to rapidly drive the field forward. Focusing on specific problems, curating challenges and datasets to address them, learning from success and failure in prospective challenges, and then curating datasets that become standard benchmarks collectively represent \emph{exactly} what we need to drive rapid progress in the field, and we've got the publications and specific success stories to prove it. How can we communicate that?]}
 %Add full list of references here.
 
 Here, we continue the legacy of SAMPL, but design a new series of SAMPL challenges specifically to maximize learning value to the community.
-Until now, this has been impossible, because SAMPL has been entirely unfunded, so its existence has required ``donation'' of data from various sources rather than data collected specifically to drive improvements to modeling. 
-Funding of SAMPL will also allow SAMPL to deliberately bridge the gap between calculations of simple physical properties like hydration -- which can be calculated fairly accurately with today's methods~\cite{mobley_blind_2014-1} -- 
-and the D3R Grand Challenges on protein-ligand binding, which are a major source of consternation for the community so far~\cite{ignjatovic_binding-affinity_2016, deng_large_2016, sunseri_d3r_2016, gathiaka_d3r_2016}.
+Until now, this has been impossible, because SAMPL has been entirely unfunded; its very existence has required ``donation'' of data and time from various sources, hindering the ability to collect dataset specifically tailored to maximize their ability to drive improvements to modeling. 
```

Comment:
Probably slightly too wordy now -- maybe "specifically tailored to drive..." is enough, not "specifically tailored to maximize their ability to drive...".

Also should be "to collect datasets", not "to collect dataset"